docs/how-to-guides/configure-dicom-server-settings.md

```diff
@@ -49,16 +49,44 @@ If you would like to update your SQL Database tier:
 ![Configure SQL2](../images/configure-sql-2.png)
 1. Click **Apply**.
 
-## Additional Configuration Settings
+## Azure Monitor
 
-**Application Insights**: By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting, navigate to your App Service:
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) offers a variety of solutions to collect, analyze and act on telemetry, including Application Insights Log Analytics.
 
+### Application Insights
+
+Application Insights is deployed when you deploy an instance of the Medical Imaging Server for DIOCM to Azure. To enable and customize Application Insights:

```

Comment:
If you just take the defaults from our template, the variable `deployApplicationInsights` is set as `true `which will turn on app insights for you. You might want to take a look and see if following these steps provides anything additional/different, and if not call out that this is already done by default.

Reply:
If I use the default, deployApplicationInsights is set as true. However, once I navigate to the application insights page on my app service, app insights is not turned on. I am given the option to 'enable application insights without redeploying your code' by 'turning on app insights'.

Reply:
@brandonpollett just wanted to follow up here!